brax/physics/colliders.py

```diff
@@ -494,7 +607,8 @@ def _collide(config: config_pb2.Config, body: bodies.Body, qp: QP,
 
   # factor of 2.0 here empirically helps object grip
   # TODO: expose friction physics parameters in config
-  return dp_n * colliding_n + dp_d * colliding_d * 2.0
+  # This factor of two makes not much of a physical sense. It kinda breaks the math, and it breaks collisions with normal not alligned with gravity vector.
```

Comment:
We have a fix for this in progress - thanks for creating the issue.  If you would like to wait for it, you can rebase after the fix is in.  It also requires changing a few hyperparams in the codebase elsewhere.

Reply:
Ok, I will leave it as is. I will fill up the issue with more precise example of the behavior that arise with this factor.